utils/sql-db-seeder/src/fee-record-payment-group/fee-record-payment-group.seeder.ts

```diff
@@ -138,12 +147,21 @@ export class FeeRecordPaymentGroupSeeder {
         requestSource: { platform: 'SYSTEM' },
       });
 
-    if (this.status === 'DOES_NOT_MATCH') {
+    if (this.status === FEE_RECORD_STATUS.DOES_NOT_MATCH) {
       const payments = paymentAmounts.map((amount) => Number((amount * 0.8).toFixed(2))).map(mapAmountToPayment);
       await dataSource.manager.save(PaymentEntity, payments);
       return;
     }
 
+    if (this.status === FEE_RECORD_STATUS.RECONCILED) {
+      const pdcReconcileUser = await MongoDbDataLoader.getPdcReconcileUserOrFail();
+      this.feeRecords.forEach((feeRecord) => {
+        feeRecord.updateLastUpdatedBy({ platform: 'TFM', userId: pdcReconcileUser._id.toString() });
+        feeRecord.dateReconciled = faker.date.recent({ days: 15 }); // eslint-disable-line no-param-reassign
```

Comment:
Where would you like this rule to be disabled then? We need to reassign a function parameter here because this is a class instance, so we can't just copy it into another instance and it is bad practice to add another method to the class just for the sake of the seeder

Reply:
I suggest looking into this kind of approach:

```ts
const modifiedRecords = this.feeRecords;

modifiedRecords.forEach((feeRecord) => {
  // ...
});

this.feeRecords = modifiedRecords;
```

And with this approach, could probably do a `.map` instead of `.forEach`

Reply:
We should never disable eslint lint rules.

Reply:
By replacing `forEach` with a `for-of` loop, the eslint disable is no longer needed.